save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -141,4 +141,12 @@ class VulnerabilityService(
             )
         }
     }
+
+    /**
+     * @param name name of project
+     * @param vulnerabilityName name of vulnerability
+     * @return Unit
+     */
+    @Transactional
+    fun deleteProject(name: String, vulnerabilityName: String) = vulnerabilityProjectRepository.deleteByNameAndVulnerabilityName(name, vulnerabilityName)
```

Comment:
Do you need `Transactional` here?

Reply:
Yes. This class is marked as _@Transactional(readOnly = true)_. Therefore, each write method must be overridden.

Reply:
Oh ok